model/metrics/deprecated/database.yaml

```diff
@@ -0,0 +1,110 @@
+groups:
+  - id: metric.deprecated.db.client.connections.count
```

Comment:
Is this id format ok? 

Alternatively `metric.db.deprecated.client.connections.count` seems also reasonable.

Reply:
We follow the <name>.deprecated pattern for attribute groups in the registry, so I guess 
`metric.db.client.connections.count.deprecated`? Either way works for me.

Reply:
Thanks. Let's follow the `.deprecated` pattern that attribute groups already use. I adapted the PR.